app_dart/lib/src/request_handlers/github_rate_limit_status.dart

```diff
@@ -35,10 +36,16 @@ class GithubRateLimitStatus extends RequestHandler<Body> {
     final Map<String, dynamic> quotaUsage = (await githubService.getRateLimit()).toJson();
     quotaUsage['timestamp'] = DateTime.now().toIso8601String();
 
+    final int remainingQuota = quotaUsage['remaining'] as int;
+    const int quotaLimitSLO = 7500; // Half of the quota limit 15000.
```

Comment:
Can we get this number programatically?

Reply:
It's in the headers and the github library does expose these.

Reply:
Updated to use the value from API data.